package/src/frontend/media/createMediaPlayer.js

```diff
@@ -0,0 +1,49 @@
+import {browser} from '../browser';
+import {VideoPlayer} from '../VideoPlayer';
+
+export const createMediaPlayer = function (options) {
+  let isAudio = options.isAudio;
+  let playsInline = options.playsInline;
+
+  let mediaElementTemplate = document.createElement(options.tagName);
```

Comment:
tagName is redundant since there is already isAudio.

Reply:
A lot of the tests also do not pass tagName. What gets created then? It might be worth covering `isAudio` with a spec.

Reply:
this problem is mainly occurred because of me having to work on two different branches going back and forth to make changes and then rebasing. let me look into it closely